test/common/common/callback_impl_test.cc

```diff
@@ -8,7 +12,7 @@ using testing::InSequence;
 namespace Envoy {
 namespace Common {
 
-class CallbackManagerTest : public testing::Test {
+class CallbackManagerTest : public testing::TestWithParam<bool> {
```

Comment:
I don't think you're using the `TestWithParam<bool>` since I see no instantiations of the parametrized test class, calls to GetParam() or use of TEST_P.

Change back to `Test`

Reply:
resolved in new PR.